### ISSUE TITLE
Crashes (maybe?) during install

### ISSUE THREAD

**Issue by 0n** [valnoxy/t480-oc#69]
When I try to install it eventually restarts and sends me back to the restore DMG, try again and it will continue over and over again

I used the regular EFI and tried installing Ventura

Also you say Thunderbolt doesn't work, should the patch to disable it always be enabled then? I don't know if that could be the issue, I left the patch disabled

My T480 differs from yours in these ways:

i7 8650u instead of an i5
32 GB instead of 8GB
Samsung 512GB SSD instead of an Intel one
it has a US keyboard instead of a German one
I have an Intel wifi card according to Ubuntu's lspci command

**Comment by Frowzy** [comment]
I'm facing the same issue. I can't seem to get past making an APFS volume, as it would simply black screen after the 40% mark in Disk Utility. I have tried cheesing it out with an HFS+ volume, however, it would get to the end of the progress bar, black screen, reboot and I'd be thrown back with an incomplete installation and I'd end up at the same restore DMG as you did.

**Comment by Frowzy** [comment]
Actually, I've been able to push it further. I've been able to get to the second stage of the installer where it's located on the SSD itself, but I have still been facing the black screen shutdown & reboots. Not sure how to diagnose these beyond the -v bootarg, which quiets out at some point in the boot process when it returns back to the apple logo and bar, then black screens and reboots about 1/4 way through, sometimes as much as 1/2. 

If I *had* to make an assumption, I'd guess it'd have to do with APFS, a driver, or a kext gone awry.

**Comment by cchewanik** [comment]
I had similar issues with my stock nvme drive. Try changing it - before buying google good brands to use with Mac, certain Samsung drives have problems with boot times

**Comment by Frowzy** [comment]
Yep. Seems I've discovered the culprit in my problem. My laptop is equipped with the Samsung PM981, which is "unsupported"

You can see it in Disk Utility, you just cannot reformat it with APFS nor finalize the installation as it'll kernel panic before you can get there. 

Has anybody had luck with Samsung OEM NVMe SSDs in the T480?

**Comment by cchewanik** [comment]
You might be able to upgrade your nvme firmware, may help not sure 

Here is a list of supported drives
https://www.tonymacx86.com/threads/new-ssds-added-to-the-recommended-list-of-the-buyers-guide.316946/

I ended up buying a WD blue SN550, has been working solid

**Comment by 0n** [comment]
> Yep. Seems I've discovered the culprit in my problem. My laptop is equipped with the Samsung PM981, which is "unsupported"
> 
> You can see it in Disk Utility, you just cannot reformat it with APFS nor finalize the installation as it'll kernel panic before you can get there.
> 
> Has anybody had luck with Samsung OEM NVMe SSDs in the T480?

You can still install MacOS on an external drive on a hackintosh like a real Mac, right?

I might try that until I buy a new SSD

**Comment by 0n** [comment]
> Yep. Seems I've discovered the culprit in my problem. My laptop is equipped with the Samsung PM981, which is "unsupported"
> 
> You can see it in Disk Utility, you just cannot reformat it with APFS nor finalize the installation as it'll kernel panic before you can get there.
> 
> Has anybody had luck with Samsung OEM NVMe SSDs in the T480?

I can now confirm it's the SSD that's the problem, I installed using an external SSD and it works perfectly normal
